votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -35,10 +35,25 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  The objective of this vote is to reduce the in-active collaborator duration
+  from 12 months to 9 months. This vote was discussed in pull-request
+  https://github.com/nodejs/node/pull/52459 and the issue #1524.
+
 
 # 6. Optionally, insert a brief introduction for the vote PR, in the markdown format.
 prBody: |
+  Currently, Node.js collaborators are considered inactive if they have not
+  contributed to the project for 12 months. The definition of contribution in
+  the context of Node.js collaborators is any approval or review of a pull
+  request, or any commit to the Node.js repository.
+  The proposed reduction in the duration of inactivity for Node.js collaborators,
```

Comment:
This text is a bit too one-sided (not introducing why some also feel that it is not a good idea). I suggest removing it from the vote intro and you can reply with it to make the case from one side.

Reply:
Removed it.